Overlay schema (#62)
moj-analytical-services/mojap-metadata: README.md

    - **description:** Description of the column.
    - **enum:** List of what values that column can take. _(Same as the standardised json schema keyword)._
    - **pattern:** Regex pattern that value has to to match (for string type_categories only). _(Same as the standardised json schema keyword)._
    - **minLength / maxLength:** The minimum and maximum length of the string (for string type_categories only). _(Same as the standardised json schema keyword)._
    - **minimum / maximum:** The minumum and maximum value a numerical type can take (for integer and float type_categories only).
- **partitions:** List of what columns in your dataset are partitions.

> - **table_location:** the location in of the table. This is optional and primarily for overlaying schemas.

Reviewer: ```suggestion
- **table_location:** the location of the table. This is a string that can represent a file path, directory, url, etc.
```

> - **database_name:**  the name of the database this table belongs to

Reviewer: ```suggestion
- **database_name:**  the name of the database this table belongs to.
```

> - **database_name:**  the name of the database this table belongs to. Optional as well for the same reason as table_location.

Reviewer: ```suggestion
- **database_name:**  the name of the database this table belongs to (optional).
```
I think this is sufficient? Atm `table_location` and `database_name` are used for the same converter (Glue) but I can that these wouldn't be so tightly coupled e.g. if you just want to read data in, why not use the `table_location` property to do so - this would not require a `database_name`. So I think it is better to not couple the properties in the docs. The docs on glue converter can state what it uses properties for.
